locales/en/common.json

```diff
@@ -1,3 +1,5 @@
 {
-  "CRYOSTAT_TRADEMARK": "Copyright The Cryostat Authors, The Universal Permissive License (UPL), Version 1.0"
+  "CRYOSTAT_TRADEMARK": "Copyright The Cryostat Authors, The Universal Permissive License (UPL), Version 1.0",
+  "EXPAND": "Expand",
+  "REMOVE": "Remove"
```

Comment:
Are these localizations still used now that there is simply a draggable edge and not a menu item?

Reply:
Nope, I will remove them, thanks.